rs/registry/canister-client/src/lib.rs

```diff
@@ -77,7 +79,20 @@ pub trait CanisterRegistryClient: Send + Sync {
         &self,
         key_prefix: &str,
         version: RegistryVersion,
-    ) -> Result<Vec<String>, RegistryClientError>;
+    ) -> Result<Vec<String>, RegistryClientError> {
+        let effective_records =
+            self.get_effective_records_between(key_prefix, RegistryVersion::new(0), version)?;
```

Comment:
Shouldn't this be "get_effective_records_between(key_prefix, version, version)?:"

It seems like you would then just get the keys.